src/gelfino/helpers.clj

```diff
@@ -0,0 +1,19 @@
+(ns gelfino.helpers
```

Comment:
Please rename this ns into common (this is the common logic between udp and tcp implementations)